aten/src/ATen/cpu/vec/vec_base.h

```diff
@@ -58,6 +58,12 @@
 #define int_vector __m256i
 #endif // CPU_CAPABILITY_AVX512
 
+#if defined(__clang__)
+#  pragma clang diagnostic push
+#  pragma clang diagnostic ignored "-Werror"
```

Comment:
ignoring warnings is not ok. We actually work hard to make this code warning free
cc @malfet

Reply:
Why do you need to touch this file btw?

Reply:
Seems like my old code was creating a bug, which requires such change, now it does not.
Let me remove, thanks for a feedback.